steam-login-startup-libgl_error-20180331.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "steam-startup-libgl_error"

```

Comment:
Can't find usage of this tag in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4749

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4748/files#diff-32797c94ffb120abddf9f3997212c570R29

Reply:
ok